src/nat/front_ends/fastapi/fastapi_front_end_plugin_worker.py

```
@@ -80,10 +92,26 @@ def __init__(self, config: Config):
+
+        if self._scheduler_address is not None:
+            if not _DASK_AVAILABLE:
+                raise RuntimeError("Dask is not available, please install it to use the FastAPI front end with Dask.")
+
+            try:
+                assert self._db_url is not None, "NAT_JOB_STORE_DB_URL environment variable must be set when using Dask"
+                self._job_store = JobStore(scheduler_address=self._scheduler_address, db_url=self._db_url)
+                self._dask_available = True
+                logger.debug("Connected to Dask scheduler at %s", self._scheduler_address)
+            except Exception as e:
+                raise RuntimeError(f"Failed to connect to Dask scheduler at {self._scheduler_address}: {e}") from e
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)